API.md

```diff
@@ -853,6 +853,41 @@ Defines whether the viewer will add padding to take account of the system status
 />
 ```
 
+#### overrideToolbarButtonBehavior
+array of [`Config.Buttons`](./src/Config/Config.ts) constants, optional, defaults to none
+
+Defines the buttons in the right section of the top app nav bar or the bottom toolbar that will skip default behavior when pressed.
```

Comment:
"Defines the option buttons in the top app nav bar or the bottom toolbar that..."?